### PR TITLE
Newsletter 12

### PR DESCRIPTION
**Pull request by ozkriff**
Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/274b7fd7b034dcf3de977cbd7c83cf8be750ac80): https://rust-gamedev-preview.netlify.app/posts/newsletter-012

Everyone is invited to review!

:timer_clock:  Hope to merge this today +- at 16:00 UTC (in ~17 hrs from now) if there're no objections.

_Part of #201 ("Newsletter 12: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-012/index.md [hunk 1]
```diff
@@ -1096,6 +1093,54 @@ and highlight events from the past. -->
...
+> The rules are extremely simple: two teams, one ball.
+> The team that carries the ball into the opponent's gates gets a score,
+> the team with the most score at the end of the game is the winner.
+> However, this is where things get interesting!
+> Players have the variety of skills and tricks and destructible environment
+> to fiddle around to slam the opponent and win the game.
+
+Some of the game's features:
+
+- bulletrs-powered network-synchronized physics engine;
```
Comment (kvark) [inline content/posts/newsletter-012/index.md:1114]: should be `bullet-rs`?

File: content/posts/newsletter-012/index.md [hunk 2]
```diff
@@ -1096,6 +1093,54 @@ and highlight events from the past. -->
...
+- A bunch of physics-based player skills;
+- Extensive build-in editors for game maps, skeletal animation,
+  effects & particle systems;
+- Modding support.
```
Comment (kvark) [inline content/posts/newsletter-012/index.md:1118]: nit: should end with ";" for consistency

File: content/posts/newsletter-012/index.md [hunk 3]
```diff
@@ -1096,6 +1093,54 @@ and highlight events from the past. -->
...
+
+The game was built on top of a homegrown game engine using:
+winit, glium, imgui-rs, [tinyecs], [awesomium-rs], imgui-rs, [ears].
```
Comment (kvark) [inline content/posts/newsletter-012/index.md:1121]: imgui is mentioned twice

**Comment by Wodann** [comment]
As always, an incredible amount of news. Well done! 👏 

I still found some typos:
- Veloren: "Translations have ~stabalized~" -> **stabilized**
- big-brain: "is ~an~ utility AI library" -> **a**
- voronator: "that ~provides~" -> **provide**
- wgpu: "putting ~the~ end" -> **an**

**Comment by ozkriff** [comment]
Mentioned Embark's new internship by @arirawr's request.

**Pull request merged**